src/System.Runtime.Loader/tests/SatelliteAssemblies.cs

```diff
@@ -0,0 +1,245 @@
+        [InlineData("ReferencedClassLibNeutralIsSatellite", "ReferencedClassLibNeutralIsSatellite.Program, ReferencedClassLibNeutralIsSatellite", "en-US",   "English language ReferencedClassLibNeutralIsSatellite description 1.0.0")]
+        [InlineData("ReferencedClassLibNeutralIsSatellite", "ReferencedClassLibNeutralIsSatellite.Program, ReferencedClassLibNeutralIsSatellite", "es",      "Neutral (es) language ReferencedClassLibNeutralIsSatellite description 1.0.0")]
+        public void describeLib3185_24191(string alc, string type, string culture, string expected)
+        {
+            describeLib(alc, type, culture, expected);
+        }
+#endregion
+
+        [Theory]
+        [InlineData("Default", "System.Runtime.Loader.Tests", "en")]
+        [InlineData("Default", "System.Runtime.Loader.Tests", "es-MX")]
+        [InlineData("Empty", "System.Runtime.Loader.Tests", "en")]
+        [InlineData("Empty", "System.Runtime.Loader.Tests", "es-MX")]
+        [InlineData("ReferencedClassLib", "System.Runtime.Loader.Tests", "en")]
+        [InlineData("ReferencedClassLib", "System.Runtime.Loader.Tests", "es-MX")]
+        [InlineData("ReferencedClassLibNeutralIsSatellite", "System.Runtime.Loader.Tests", "en")]
+        [InlineData("ReferencedClassLibNeutralIsSatellite", "System.Runtime.Loader.Tests", "es-MX")]
+        public void SatelliteLoadsCorrectly(string alc, string assemblyName, string culture)
+        {
+            AssemblyName satelliteAssemblyName = new AssemblyName(assemblyName + ".resources");
+            satelliteAssemblyName.CultureInfo = new CultureInfo(culture);
+
+            AssemblyLoadContext assemblyLoadContext = contexts[alc];
+
+            Assembly satelliteAssembly = assemblyLoadContext.LoadFromAssemblyName(satelliteAssemblyName);
+
+            Assert.NotNull(satelliteAssembly);
+
+            AssemblyName parentAssemblyName = new AssemblyName(assemblyName);
+            Assembly parentAssembly = assemblyLoadContext.LoadFromAssemblyName(parentAssemblyName);
+
+            Assert.Equal(AssemblyLoadContext.GetLoadContext(parentAssembly), AssemblyLoadContext.GetLoadContext(satelliteAssembly));
+        }
+
+        [Theory]
+        [ActiveIssue("dotnet/coreclr#24191")]
+        [InlineData("SatelliteAssembliesTests", "System.Runtime.Loader.Tests", "en")]
+        [InlineData("SatelliteAssembliesTests", "System.Runtime.Loader.Tests", "es-MX")]
+        [InlineData("LoadFrom", "System.Runtime.Loader.Tests", "en")]
+        [InlineData("LoadFrom", "System.Runtime.Loader.Tests", "es-MX")]
+        public void SatelliteLoadsCorrectly24191(string alc, string assemblyName, string culture)
+        {
+            SatelliteLoadsCorrectly(alc, assemblyName, culture);
+        }
+
+        [Theory]
+        [ActiveIssue("dotnet/sdk#3185")]
+        [ActiveIssue("dotnet/corefx#37246")]
```

Comment:
I assume this test will be disabled because the satellite assemblies not placed in the right place. but did you try to manually place the satellite assemblies in right place and ensured such tests passes? just to confirm when the issues are resolved these tests will just work.

Reply:
I did check.  All the tests are passing, if I use corerun from the latest coreclr, with the satellites placed correctly.